src/formatting/overflow.rs

```diff
@@ -588,6 +589,13 @@ impl<'a> Context<'a> {
 
     fn rewrite_items(&self) -> Option<(bool, String)> {
         let span = self.items_span();
+        if self.items.len() == 0 && contains_comment(self.context.snippet(span)) {
+            let hi = self.context.snippet_provider.span_before(span, self.suffix);
+            let cmnt =
+                rewrite_missing_comment(mk_sp(span.lo(), hi), self.nested_shape, self.context)?;
```

Comment:
nit:
```suggestion
                rewrite_missing_comment(span.to(hi), self.nested_shape, self.context)?;
```

Reply:
As per the [doc](https://docs.rs/rustc-ap-rustc_span/712.0.0/rustc_ap_rustc_span/struct.Span.html#method.to) `to` method is taking a `Span` as the first argument. But `hi` is a `BytePos`. So I used the [with_hi](https://docs.rs/rustc-ap-rustc_span/712.0.0/rustc_ap_rustc_span/struct.Span.html#method.with_hi) method.